airbyte-integrations/base-python/airbyte_protocol/entrypoint.py

```diff
@@ -65,8 +66,8 @@ def start(self, args):
 
         with tempfile.TemporaryDirectory() as temp_dir:
             if cmd == "spec":
-                # todo: output this as a JSON formatted message
-                print(self.source.spec(logger).spec_string)
+                message = AirbyteMessage(type='SPEC', spec=self.source.spec(logger))
```

Comment:
@jrhizor why does this take a logger?

Reply:
I did it so it is consistent with the other methods

Reply:
ok